docs/concepts_function_scope_example.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
```

Comment:
I don't care if we make this import for test files, but I don't think we should import this in the files in _docs._
The Python scripts in the _docs_ folder get included as snippets in the documentation. The future import adds noise to the snippet and may be confusing.

Reply:
```suggestion
```

Reply:
Right, fixed :D